<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppFileManagerGUIClass</class>
 <widget class="QMainWindow" name="CppFileManagerGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CppFileManagerGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Szukaj:</string>
      </property>
      <property name="buddy">
       <cstring>fileLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Adres:</string>
      </property>
      <property name="buddy">
       <cstring>pathLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pathLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Zawartość:</string>
      </property>
      <property name="buddy">
       <cstring>fileList</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="fileList">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createFolderButton">
      <property name="text">
       <string>Stwórz Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createFileButton">
      <property name="text">
       <string>Stwórz Plik</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openFolder">
      <property name="text">
       <string>Otwórz bieżący katalog</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CppFileManagerGUI.qrc"/>
 </resources>
 <connections/>
</ui>
